### ISSUE TITLE
 Support for RFC 6868

### ISSUE THREAD

**Issue by rfc2822** [ical4j/ical4j#83]
Double-quotes can't be encoded in plain iCalendar files. This is why RFC 6868 specifies a way to encode " (and \n) using the newly introduced ^ escape character.

It might be worth a thought whether RFC 6868 could be implemented in ical4j (and maybe enabled by default, and/or enabled/disabled by a configuration option). At the moment, I don't know any platforms implementing RFC 6868, but using RFC 6868 is the only useful solution for fields with double-quotes I know.

Thanks for your great work.

**Comment by benfortuna** [comment]
Yes I think this should be possible, but initially will try to make it an optional feature in case of unintended side-effects. An interesting choice of escape character..

https://tools.ietf.org/html/rfc6868
